.gitchangelog.rc

```diff
@@ -82,16 +82,22 @@ ignore_regexps = [
 ## whenever you are tweaking this variable.
 ##
 section_regexps = [
-    ('New', [
-        r'^[nN]ew\s*:\s*((dev|use?r|pkg|test|doc)\s*:\s*)?([^\n]*)$',
-     ]),
-    ('Changes', [
-        r'^[cC]hg\s*:\s*((dev|use?r|pkg|test|doc)\s*:\s*)?([^\n]*)$',
-     ]),
     ('Fix', [
-        r'^[fF]ix\s*:\s*((dev|use?r|pkg|test|doc)\s*:\s*)?([^\n]*)$',
-     ]),
+        r'''(?ix)
+            (^fix)
+            |
+            (^\[fixt)
+        ''',
+    ]),
+    ('Feature', [
+        r'''(?ix)^
+            (
+                (\[?feature)
+                |
+                (\[?KLTB00)
```

Comment:
This is currently just a placeholder for Jira task extraction (not fully implemented yet). I'll add `KLTB002` as default value :+1: 
It can be configured by overriding `.gitchangelog.rc` in project/git root directoy.

Reply:
Updated default to `KLTB002`